themes/digital.gov/layouts/topics/terms.html

```diff
@@ -3,74 +3,54 @@
 {{- define "content" -}}
   <main role="main" id="main-content">
     <header>
-      <div class="grid-container grid-container-desktop">
+      <div class="grid-container grid-container-desktop margin-top-2">
```

Comment:
Added `margin-top-2` for header spacing.

Reply:
thakn you for doing that!